shared/js/dataDriver.js

```diff
@@ -343,11 +359,17 @@ loop.DataDriver = function() {
       // Determine the clock skew before continuing initialization.
       this.update("meta", "lastConnect").then(({ timestamp }) => {
```

Comment:
I split out the code starting from Promise.all as the "update" for lastConnect and setting `_clockSkew` really are part of connecting. The update is just how dataDriver reads out the server time, but it doesn't actually update room data.